src/app/ui/TaskBoard.tsx

```diff
@@ -129,9 +129,11 @@ export const TaskBoard = ({
         >
           <NewTaskForm
             handleCreate={() => {
-              store.dispatch(setShowModal())
-              store.dispatch(clearCreateTaskFields())
-              handleCreate({ title, body: description, workflowStateId, assigneeType, assigneeId })
+              if (title) {
+                store.dispatch(setShowModal())
+                store.dispatch(clearCreateTaskFields())
+                handleCreate({ title, body: description, workflowStateId, assigneeType, assigneeId })
```

Comment:
Please also add `title: z.string().min(1)` logic to the corresponding CreateTaskRequestSchema and parse using it

Reply:
made changes in the commit below